.env.example

```diff
@@ -5,6 +5,9 @@ NEXT_PUBLIC_MATOMO_URL="https://piwik.technologiestiftung-berlin.de"
 NEXT_PUBLIC_MATOMO_SITE_ID="13"
 NEXT_PUBLIC_BASE_URL="https://localhost:3000"
 
+# for requests to the giessdenkiez.de API
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0
```

Comment:
Just a detail, but how about we name these new env vars `GDK_API_ANON_KEY` and `GDK_API_URL`? Just to communicate clearly that this is an external API and not a Supabase for Baumblick?

Reply:
Yes. Fixed.